# Akila Ganlath

# Joint States Publishers -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# State Controllers ---------------------------------------
left_rear_wheel_velocity_controller:
  type: effort_controllers/JointVelocityController
  pid: {p: 1.0, i: 0.0, d: 0.0, i_clamp: 0.0}

right_rear_wheel_velocity_controller:
  type: effort_controllers/JointVelocityController
  pid: {p: 1.0, i: 0.0, d: 0.0, i_clamp: 0.0}

left_front_wheel_velocity_controller:
  type: effort_controllers/JointVelocityController
  pid: {p: 1.0, i: 0.0, d: 0.0, i_clamp: 0.0}

right_front_wheel_velocity_controller:
  type: effort_controllers/JointVelocityController
  pid: {p: 1.0, i: 0.0, d: 0.0, i_clamp: 0.0}

left_steering_hinge_position_controller:
  type: effort_controllers/JointPositionController
  pid: {p: 50.0, i: 0.0, d: 0.5}

right_steering_hinge_position_controller:
  type: effort_controllers/JointPositionController
  pid: {p: 50.0, i: 0.0, d: 0.5}
